src/Components/Common/components/ResponsiveProcedureTables.tsx

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useState } from "react";
+import AccordionV2 from "./AccordionV2";
+
+function getWindowSize() {
+  const { innerWidth, innerHeight } = window;
+  return { innerWidth, innerHeight };
+}
+
+export default function ResponsiveProcedureTable(props: {
+  theads: Array<string>;
+  list: Array<any>;
+  objectKeys: Array<any>;
+  fieldsToDisplay: Array<number>;
+}) {
+  const [windowSize, setWindowSize] = useState(getWindowSize());
+  useEffect(() => {
+    function handleWindowResize() {
+      setWindowSize(getWindowSize());
+    }
+
+    window.addEventListener("resize", handleWindowResize);
+
+    return () => {
+      window.removeEventListener("resize", handleWindowResize);
+    };
+  }, []);
+  return (
+    <>
+      {windowSize.innerWidth > 1024 ? (
```

Comment:
I don't see a reason to have this much complication. Wouldn't tailwind's modifiers be enough?

```suggestion
  useEffect(() => {
    function handleWindowResize() {
      setWindowSize(getWindowSize());
    }

    window.addEventListener("resize", handleWindowResize);

    return () => {
      window.removeEventListener("resize", handleWindowResize);
    };
  }, []);
  return (
    <>
      {windowSize.innerWidth > 1024 ? (
```